_projects/2024/sunpy/eclipse.md

```diff
@@ -0,0 +1,68 @@
+---
+name: Solar Eclipse analysis with sunpy
+desc: Process and analysis images taken of the total solar eclipse that crosses the US in 2024
+requirements:
+ - Experience with Python
+ - Background with Physics or Astronomy
+difficulty: very high
+issues:
+ - https://github.com/sunpy/sunpy/issues/7387
+mentors:
```

Comment:
Someone with coronagraph analysis background would be very helpful if not someone that explicitly has analyzed eclipse images. I know there are a number of folks at Goddard that have traveled and work on eclipse data. Alex Young could be a good person to ask to suggest someone. I know that Natchimuthuk Gopalswamy has been to a number of eclipses.

Reply:
If you can ask, that would be great